front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion.tsx

```diff
@@ -38,21 +33,22 @@ const FormResultsQuestion = ({
   inputType,
   answers,
   totalResponses,
+  totalSubmissions,
   required,
   customFieldId,
   textResponses = [],
+  files = [],
 }: FormResultsQuestionProps) => {
   const isAnalysisEnabled = useFeatureFlag({ name: 'analysis' });
   const { formatMessage } = useIntl();
 
-  // TODO: Replace hardcoded '2' here. Urgent to relese to fix bug though right now.
```

Comment:
Notice that you have made the same refactor on your image question branch so I had a number of merge conflicts to resolve when I merged this into randomisation!